src/com/synopsys/integration/pipeline/SimplePipeline.groovy

```diff
@@ -50,6 +50,18 @@ class SimplePipeline extends Pipeline {
         this.commonRunDirectory = commonRunDirectory
     }
 
+    static SimplePipeline commonPipeline(CpsScript script, String branch, String relativeDirectory, String url) {
```

Comment:
You may want to capitalize the method name since this is a static method. COMMON_PIPELINE

Reply:
@jamesrichard91 Done!